.eslintrc.js

```diff
@@ -33,5 +33,6 @@ module.exports = {
     "import/no-cycle": "error",
     "no-unused-vars": "off",
     "prefer-template": "error",
+    "endOfLine": "auto",
```

Comment:
So the problem on my side isn't solvable with `.gitattributes` `text=auto`, because it's not about what is gonna be committed at the end (my git properly converts everything to `lf` during push), but it's about windows auto-converting lines to `crlf` on retrieval, which automatically puts on fire all of the files since prettier tells me to change from `crlf` to `lf`. This rule solves the problem, and I'm still pushing `lf` to the repo with no issues.

Reply:
I still don't see this eslint rule. Where are you getting all these eslint rules from? Also, please insert the rule in alphabetical order.

Reply:
https://stackoverflow.com/a/53769213/2743158 - it's prettier rule, which can be put into eslint config essentially

Reply:
This needs to be under the prettier rule of eslint rules. Please use `lf` instead of `auto` as that would normalize the whole file by following the line ending of the first line.

Or to make it simpler, just use eslint's `linebreak-style: ["error", "unix"]`. https://eslint.org/docs/rules/linebreak-style

Also, you can run the linter through `yarn lint` in the sdk folder.

Reply:
Found solution without the rule on my side, removed the rule.